language/move-lang/tests/move_check/borrows/assign_local_combo_invalid.exp

```diff
@@ -1,55 +1,50 @@
 error: 
 
-    ┌── tests/move_check/borrows/assign_local_combo_invalid.move:14:9 ───
+    ┌── tests/move_check/borrows/assign_local_combo_invalid.move:13:37 ───
     │
- 14 │         s = S { f: 0, g: 0 };
-    │         ^ Invalid assignment of local 's'
-    ·
  13 │         if (cond) f = &s.f else f = &s.g;
     │                                     ---- It is still being borrowed by this reference
+ 14 │         s = S { f: 0, g: 0 };
+    │         ^ Invalid assignment of local 's'
```

Comment:
These error messages got *a lot* worse now that the primary label isn't printed first. If there is no way of fixing this, we will probably have to stay at the old version, or in the worst case, fork `codespan-reporting`

Reply:
agreed about the primary item not printed first, I'll poke around the emit method more.

Reply:
If I can't find a way though, I'd rather not take on forking codespan-reporting - I've only attempted the upgrade because the dependabot pr has sat for week and wanted to try to close it out.

Reply:
I'll go open an issue on their github. Let's delay this for now.

Reply:
Actually, maybe this looks less bad with the colors? Do you have a screenshot with the color version?

Reply:
@tnowacki Update all items you had issues with in coding style (and replicate the fixes to the move-prover as well).    Based on the response from codespan authors on the bug it looks like the recommendation is update our error messages.  Not sure the best way to run one of the move-lang's compiles locally, happy to give you a screenshot of their color choices if you let me know how to produce it.

Reply:
put the fixes in this commit https://github.com/libra/libra/pull/3447/commits/067aaf8de5375451e6569aaec7e56a3f184215cc  (separate from the rebased original code changes), https://github.com/libra/libra/pull/3447/commits/65bf5d7fd9e8202b64e7d55bae9a89c3ccf52463 happy to merge them if you'd prefer to read that way.